packages/realm-web/src/index.test.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from "chai";
+
+import * as Realm from ".";
+
+describe("Realm Web", () => {
+    it("expose the App constructor", () => {
+        expect(typeof Realm.App).to.equal("function");
+    });
+
+    describe("Credentials", () => {
+        it("expose a credentials factory", () => {
```

Comment:
I think we should not be doing this or we should refactor how these are exposed. For example having a `Realm.CredentialsFactory.anonymous()` or `Realm.CommonCredentials.anonymous` or something like that.

Reply:
Can you provide a reasoning behind that opinion? As mentioned above it's a pattern we already use.

I personally like giving users simple shorthands for something like this as it makes it easier to discover the API and has very little costs (runtime efficiency, maintenance, ...) on our side. I see little value in this as a shorthand if we used names that are harder to guess / discover.

Reply:
This was discussed in a Zoom meeting. I've pushed an update with the agreed changes, please re-review that part :)